[UI] Update the theme of light/dark mode

ChengYanJin opened the pull request:
**Component**: ui, theme

**Context**: 
The current theme is not up to date with the design. There are some variables missing too.

**Summary**:
- Update the dark mode with the final version
- Update the status color of light mode with the proposal, but keep the primary/text/border as the previous version, since it doesn't work well.

**Acceptance criteria**: 
For the frontent devs to use the correct variable name to align with the design. 
We may want to slightly change some colors later.

ChengYanJin committed:
ui/theme: Update the them for light/dark mode

- Update the dark mode with final version
- Update the status color of light mode with proposal, but keep the primary/text/border as the previous version,
  since it doesn't work well.

bert-e commented:
# Hello chengyanjin,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `improvement/update-the-theme-with-design` into
`development/2.6`
* [w/2.7/improvement/update-the-theme-with-design](https://github.com/scality/metalk8s/compare/development/2.7...w/2.7/improvement/update-the-theme-with-design) will be merged into `development/2.7` 



The following branches will **NOT** be impacted:

* `development/1.0`
* `development/1.1`
* `development/1.2`
* `development/1.3`
* `development/2.0`
* `development/2.1`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @tcarmet

* @nicolast

* @lucieleonard

* @alexandre-allard-scality

* @monpote

* @slaperche-scality

* @teddyandrieux

* @cmonfort

* @thomasdanan

* @carlito-scality

* @jbertran

* @sayf-eddine-scality

* @gdemonet

* @nootal

ChengYanJin committed:
ui/salt: Update the theme.json in configMap

Refs: #2777

bert-e commented:
# History mismatch



Merge commit #78648c53ee997906effec0f94763ee5b612c0d9e on the integration branch
`w/2.7/improvement/update-the-theme-with-design` is merging a branch which is neither the current
branch `improvement/update-the-theme-with-design` nor the development branch
`development/2.7`.

It is likely due to a rebase of the branch `improvement/update-the-theme-with-design` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

gdemonet committed:
build,salt: Render UI branding ConfigMap

We used to copy-paste the contents of the ui/public/brand/theme.json
file to this ConfigMap manifest.
To make sure we never forget about it nor make any mistake when copying,
we switch to templating this manifest from the contents of the original
file.
